packages/react-form-state/src/components/tests/Nested.tsx

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import faker from 'faker';
+import {mount} from 'enzyme';
+
+import {Input} from './components';
+import FormState from '../..';
+
+const ARBITRARY_SEED = 1337;
+
+describe('<Nested />', () => {
+  beforeEach(() => {
+    faker.seed(ARBITRARY_SEED);
+  });
+
+  it('passes field state into child function', () => {
+    const renderPropSpy = jest.fn(() => null);
+
+    const product = {
+      title: faker.commerce.productName(),
```

Comment:
First time that I see `faker`, do we have access to it in `web`?

Reply:
yes!

Reply:
It's pretty handy.